lib/plugins/aws/provider/awsProvider.js

```diff
@@ -140,6 +140,11 @@ class AwsProvider {
 
       // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8016
       serverless.configSchemaHandler.defineProvider('aws', {
+        provider: {
+          properties: {
+            resourcePolicy: { type: 'array' },
```

Comment:
Maybe we can make it a bit more precise? If I see correctly each item should be an object, therefore maybe let's add `items: { type: 'object' }`